en/about/index.md

```diff
@@ -170,10 +170,10 @@ Ruby has a wealth of other features, among which are the following:
 
 * Ruby features OS independent threading. Thus, for all platforms on
   which Ruby runs, you also have multithreading, regardless of if the OS
-  supports it or not, even on MS-DOS!
+  supports it or not.
```

Comment:
Hi.
The current ruby (maybe from 1.9 and later) depend on pthread/WinThread to implement Thread.
The whole description in this section is totally false for recent ruby and could be removed.

Reply:
Ok, that's fine with me. It was already gone from the French version. I removed it from all the others.